Heavily minimized version of “play sfx when level up” by sergi4ua <sergiy4ua@gmail.com>



Rebased on devilutionX 0.3.1.

diff --git a/Source/player.cpp b/Source/player.cpp
index aa89dbc..3bc858c 100644
--- a/Source/player.cpp
+++ b/Source/player.cpp
@@ -741,6 +741,8 @@ void __fastcall NextPlrLevel(int pnum)
 	if (pnum == myplr) {
 		drawmanaflag = TRUE;
 	}
+
+	PlaySFX(IS_QUESTDN);
 }
 
 void __fastcall AddPlrExperience(int pnum, int lvl, int exp)
